fog-clearer/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fog Clearer</title>
+</head>
+
+<body>
+    <script>
+        function fogClearer(stringToClear) {
+            var fog = "";
+            for (var i = 0; i < stringToClear.length; i++) {
+                if (stringToClear[i] !== "f" && stringToClear[i] !== "o" && stringToClear[i] !== "g" && stringToClear.indexOf("f") > -1) {
+                    fog += stringToClear[i];
+                } else if (stringToClear.indexOf("f") === -1) {
```

Comment:
Dont forget you must also look for o and g. just looking for f won't work if you receive an argument with os and gs!